src/System.Security.Permissions/src/System/Security/HostProtectionException.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace System.Security
+{
+    public partial class HostProtectionException : System.SystemException
+    {
+        public HostProtectionException() { }
+        // protected HostProtectionException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        public HostProtectionException(string message) { }
+        public HostProtectionException(string message, System.Exception e) { }
+        public HostProtectionException(string message, System.Security.Permissions.HostProtectionResource protectedResources, System.Security.Permissions.HostProtectionResource demandedResources) { }
+        public System.Security.Permissions.HostProtectionResource DemandedResources { get { return default(System.Security.Permissions.HostProtectionResource); } }
+        public System.Security.Permissions.HostProtectionResource ProtectedResources { get { return default(System.Security.Permissions.HostProtectionResource); } }
+        // public override void GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        public override string ToString() { return default(string); }
```

Comment:
In general, let's not override ToString unless we have something real to do.  The base Exception.ToString implementation provides a lot of value, including outputting the exception's stack trace.  While we don't expect this particular Exception-derived type to really be used, if someone does choose to throw it, we should have it behave as most exceptions would.